astrocut/asdf_cutouts.py

```diff
@@ -16,40 +16,42 @@
 
 
 def _get_cloud_http(s3_uri: str) -> str:
-    """ Get the HTTP URI of a cloud resource from an S3 URI
+    """ 
+    Get the HTTP URI of a cloud resource from an S3 URI.
 
     Parameters
     ----------
     s3_uri : string
         the S3 URI of the cloud resource
     """
     # create file system
-    fs = s3fs.S3FileSystem(anon=True)
+    fs = s3fs.S3FileSystem()
```

Comment:
This change was to resolve an issue in RSP. Though not directly related to documentation, I thought I would include it in this PR since it's a small fix.